catlas/nuclearity.py

```diff
@@ -65,12 +65,11 @@ def get_nuclearity_neighbor_counts(surface_atoms_of_element, connectivity_matrix
     """
     Function that counts the like surface neighbors for surface atoms
     Args:
-        surface_atoms_of_element: list of all surface atoms which are of a specific
-        element
-        connectivity_matrix: matrix memorializing which atoms in the slab are connected
+        surface_atoms_of_element (ase.Atoms): list of all surface atoms which are of a specific element
+        connectivity_matrix (numpy.ndarray[int8]): which atoms in the slab are connected
```

Comment:
Do you happen to know if the `type` of connectivity/nuclearity objects are correct as listed? I got lost in documentation and made an educated guess.

Reply:
definitely the first is wrong. As the description includes it is a list